modules/fedimint-ln-client/src/lib.rs

```diff
@@ -903,7 +904,10 @@ impl LightningClientModule {
             .get_value(&MetaOverridesKey {})
             .await
         {
-            if meta.fetched_at.elapsed().unwrap() < META_OVERRIDE_CACHE_DURATION {
+            let elapsed = now()
+                .duration_since(meta.fetched_at)
+                .expect("time is always increasing");
```

Comment:
That is not technically correct for `SystemTime`:

![Screenshot 2024-02-20 at 20-05-21 SystemTime in std time - Rust](https://github.com/fedimint/fedimint/assets/84478054/313614db-0aee-4745-8a22-760cc2f5195d)

Maybe an `unwrap_or` would be better?

Reply:
Did `unwrap_or(0)` so it'll just refetch

Reply:
I believe `unwrap_or(0)` (or `unwrap_or_default` in merged diff) will return from the cache instead of refetching. I think either works, but want to highlight incase you specifically wanted the behavior to refetch.